packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -171,6 +174,13 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
   /// If null then [kMaxFlingVelocity] is used.
   double maxFlingVelocity;
 
+  /// The velocity tracker [addAllowedPointer] should use to track a new pointer,
```

Comment:
Since this will be the first place a user is exposed to the term "velocity tracker", describe what it is or reference another doc